a8cteam51/safety-net
# Allow password reset emails

## Synopsis
Simplified the 'stop emails' functionality considerably. Now I'm just filtering `wp_mail` and removing the `to` field from any email that isn't a password reset email.

Also was able to get rid of the other very complicated class.

## To test
1. Try sending password reset email - should work
2. Try sending any other email - should be blocked, and a "Blocked outgoing email" line added in the debug.log
